index.html

```diff
@@ -81,6 +81,209 @@ <h1 class="heading-primary">
+                Gain+Glory
+              </h3>
+              <ul class="work-features">
+                <li class="work-feature-item">Ruby on Rails</li>
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+          <!-- Work 1 -->
+          <div class="work">
+            <div class="work-image-holder"></div>
+            <div class="work-details">
+              <h3 class="heading-tertiary">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h3>
+              <ul class="work-features">
+                <li class="work-feature-item">Ruby on Rails</li>
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!----------------------->
+      <!-- ABOUT ME SECTION -->
+      <!----------------------->
+
+      <section class="section-about-me">
+        <div class="about-me-header">
+          <h2 class="heading-secondary">About me</h2>
+          <p class="about-me-description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <a href="#" class="btn">Get my resume</a>
+          <img
+            class="about-me-top-bg"
+            src="imgs/about-me-top.png"
+            alt="Beautiful illustration"
+          />
+          <img
+            class="about-me-bottom-bg"
+            src="imgs/about-me-bottom .png"
+            alt="Beautiful illustration"
+          />
+        </div>
+
+        <!-- CARDS -->
+        <ul class="cards">
+          <!-- Languages card -->
+          <li class="card">
+            <img
+              src="imgs/card-icons/icon-languages.png"
+              alt="Language Icon"
+              class="card-icon"
+            />
+            <h3 class="heading-tertiary">Languages</h3>
+            <ul class="flex card-items">
+              <li class="card-btn">JavaScript</li>
+              <li class="card-btn">Ruby</li>
+              <li class="card-btn">HTML</li>
+              <li class="card-btn">CSS</li>
+            </ul>
+          </li>
+          <!-- Framworks card -->
+          <li class="card">
+            <img
+              src="imgs/card-icons/icon-frameworks.png"
+              alt="Frameworks Icon"
+              class="card-icon"
+            />
+            <h3 class="heading-tertiary">Frameworks</h3>
+            <div class="card-items-conainer">
```

Comment:
- [ ] Thank you so much for implementing changes requested from your previous reviewer that looks so good,but i observed some broken images on your UI ,Kindly fix this 
- [ ] I also notice that your background images not display too please kindly fix it to make  you website look exactly like original design on figma template.
- [ ] your design
![figmwww](https://user-images.githubusercontent.com/92458236/235158735-47d79535-7b37-4f6e-87b8-f12fdaf2b700.jpg)
- [ ] Your design 
![figm ww](https://user-images.githubusercontent.com/92458236/235158928-174cc5e6-d36d-45ed-808c-9d76e8e47d0b.jpg)
   - [ ] Figma design
![figma templ3](https://user-images.githubusercontent.com/92458236/235159081-bd6e0363-c677-443d-9f30-d941ac8b39d5.jpg)

Reply:
Hey @ademibowale.

I think there might be some problems with yours side. for me , it works perfect.

Could You please re-check it. @ademibowale .

Reply:
![image](https://user-images.githubusercontent.com/131291114/235167964-3f1f46a2-b54c-40f3-bd30-5ce8150e606d.png)
![image](https://user-images.githubusercontent.com/131291114/235168048-667440d7-ebf9-4155-a878-a61d4187a94a.png)
![image](https://user-images.githubusercontent.com/131291114/235168142-aba78024-f147-48af-acaa-a3df8283f69d.png)
![image](https://user-images.githubusercontent.com/131291114/235168220-9536ca76-9878-463a-bffb-fac0ae188c0e.png)

Reply:
It's the "add-works-and-aboutme-sections" branch.